inc/saiobject.h

```diff
@@ -37,6 +37,7 @@
 #include <saisrv6.h>
 
 /* new experimental object type includes */
+#include <saiexperimentaldashha.h>
```

Comment:
this change is not needed

Reply:
thanks for checking Kamil! I have it removed now.